packages/contentstack-import/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@contentstack/cli-cm-import",
   "description": "Contentstack CLI plugin to import content into stack",
-  "version": "1.11.0",
+  "version": "1.11.1",
```

Comment:
Pls verify the version before merging it into development branch

Reply:
This version bump is good.